clang/test/SemaSYCL/sycl-external-attribute.cpp

```diff
@@ -0,0 +1,52 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -DSYCL %s
+// RUN: %clang_cc1 -fsycl-is-host -fsyntax-only -verify -DHOST %s
```

Comment:
Since SYCL applies to both host and device, let's differentiate by "host" and "device" (with or without "SYCL")
```suggestion
// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify -DDEVICE %s
// RUN: %clang_cc1 -fsycl-is-host -fsyntax-only -verify -DHOST %s
```